website_sale_partner_restrict_payment_acquirer/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+Open the partner form and go to the `Sales & Purchases` tab. Add payment acquirers in the `Allowed Acquirers` field.
+When the partner proceeds to checkout in the portal, only the allowed acquirers will be available for payment.
+
+Note: Leaving this field empty will remove restrictions and allow the selection of any acquirer.
+
+For not logged-in users, if you want to restrict their allowed acquirers, you need to edit the `Allowed Acquirers` field for the `Public user` partner (ID=4).
```

Comment:
Although in most case ID will be 4, this is not granted. So one should never use fixed IDs. If you need to reference some particular record, please use external reference instead.